openapi.yaml

```diff
@@ -4,9 +4,7 @@ info:
   description: API for Heart of the Valley
   version: 0.0.1
 servers:
-  - url: http://localhost:8080
-    description: Development Server
-  - url: https://st4s0w4in7.execute-api.us-west-2.amazonaws.com
+  - url: https://s06hks51dl.execute-api.us-west-2.amazonaws.com
```

Comment:
should this also be an env var?

Reply:
The openapi.yaml file acts as the single source of truth for all codegen in the project. This helps the backend and frontend maintain a common contract. Unfortunately yaml is not a programming language (so much missed opportunity) and so no env vars

Reply:
but it is safe to keep in here?